tests/conftest.py

```diff
@@ -5,7 +5,7 @@
 from packaging import version
 
 SNAPSHOT_MINIMUM_VERSIONS = {
-    "textual": "0.81",
+    "textual": "0.87.1",
```

Comment:
We should update this to whatever version their next release is (either 0.88.0 or 0.87.2), since we know these tests don't pass with 0.87.1 (they need a release that contains https://github.com/Textualize/textual/pull/5296 to pass)